INSTALL.md

```diff
@@ -95,13 +95,13 @@ test -f matrix_key.pem || ./bin/generate-keys -private-key matrix_key.pem
 
 Create config file, based on `dendrite-config.yaml`. Call it `dendrite.yaml`. Things that will need editing include *at least*:
 * `server_name`
-* `database/*`
+* `database/*` (All lines in the database section must have the password of the postgres db running in your system. eg:```dendrite:password@localhost``` )
 
 
 ## Starting a monolith server
 
 It is possible to use 'naffka' as an in-process replacement to Kafka when using
-the monolith server. To do this, set `use_naffka: true` in `dendrite.yaml`.
+the monolith server. To do this, set `use_naffka: true` in `dendrite.yaml` and uncomment the necessary line related to naffka(last line in the *database* section) to use naffka.
```

Comment:
Looks like you left in a bit of extra words here? (And there should be a space between word and `(`). Also naffka might not necessarily always be the last line in `database`. I'd suggest something along the lines of the following:

```suggestion
the monolith server. To do this, set `use_naffka: true` in `dendrite.yaml` and uncomment
the necessary line related to naffka in the `database` section. Be sure to update the
database username and password if needed.
```

Reply:
Done.